src/dotnet/UI.Blazor/Services/EscapeHandler.cs

```diff
@@ -0,0 +1,38 @@
+using ActualChat.UI.Blazor.Module;
+
+namespace ActualChat.UI.Blazor.Services;
+
+public sealed class EscapeHandler : IAsyncDisposable
```

Comment:
So I'd rather turn this into a "headless" scoped service like ModalUI or ClipboardUI, which attaches the handler to HTML `body` element rather than affects rendering in any way.